pkg/document/time/ticket.go

```diff
@@ -18,6 +18,13 @@ package time
 
 import (
 	"fmt"
+	"math"
+)
+
+const (
+	less = iota - 1
+	equal
+	greater
```

Comment:
Is this a popular pattern in Go? I haven't seen much of this pattern in the Golang base package.

Reply:
Yes. example, https://golang.org/src/sync/mutex.go?s=765:813#L36
If you want to check more, recommended to try `rg iota ** / *. go` or `grep iota ** / *. go` in `go/src` directory.

Reply:
I've seen many Go base packages use 1, 0, -1 instead of less, equal, greater. Is this pattern more readable?

Reply:
My answer is "yes". However, I agree that it is good to follow base package. I'll fix it